$MirOS: ports/comms/linphone/patches/patch-config_h_in,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- config.h.in.orig	Wed Oct  6 00:29:10 2004
+++ config.h.in	Wed Oct  6 00:29:23 2004
@@ -128,8 +128,5 @@
 /* Defined if we are compiling for arm processor */
 #undef __ARM__
 
-/* Defined if we are compiling for FreeBSD */
-#undef __FreeBSD__
-
 /* Defined if we are compiling for linux */
 #undef __LINUX__
